docs/tools/curl.md

```diff
@@ -31,12 +35,15 @@ curl \
   $WEBHOOK_URL
 ```
 
-### PowerShell
+### PowerShell 5.1 and below (powershell.exe)
```

Comment:
can we have single section for powershell as except of `curl` being `iwr` alias it's fully interchangeable if using `curl.exe`?